### PR TITLE
Changes in Record and Achievement

### PR DESCRIPTION
**Pull request by fkrdud1125**
- ReadMe Update
- Achievement renewal conditions limited to single player play
- Achievement details changed (combo achievement added, 2-player achievement removed -> achievement requiring more than 10 minutes of play added)
- Improved hit rate logic (hitbullets variable added when hitting an enemy ship or item)
- Minor bug fixes in addition to the above

### CODE REVIEW

**Review by brewcoua — APPROVED** [review]
LGTM :+1:

File: src/entity/Web.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import java.awt.*;
 
-public class Web extends Entity{
+public class  Web extends Entity{
```
Comment (dev-moonsc) [inline src/entity/Web.java:7]: Why this line changed? I think it's unnecessary changes.

**Review by dev-moonsc — APPROVED** [review]
LGTM :)

File: src/entity/Web.java (hunk 1 above)
Comment (fkrdud1125) [reply src/entity/Web.java:7]: A typo has been made. We'll change it back the next time we merge! 👍
